tests/testsuite/new.rs

```diff
@@ -472,6 +472,19 @@ or change the name in Cargo.toml with:
 fn git_default_branch() {
     // Check for init.defaultBranch support.
     create_empty_gitconfig();
+
+    // If we're running this under a user account that has a different default branch set up
+    // this test will fail. We avoid that setting the defaultBranch and then ensuring that
+    // the changed setting is respected.
+    fs::write(
+        paths::home().join(".gitconfig"),
+        r#"
+        [init]
+            defaultBranch = master
+        "#,
+    )
+    .unwrap();
```

Comment:
I guess these lines can be merged into `create_empty_gitconfig`. Though the function itself needs a rename.

Reply:
Good idea, done!